### ISSUE TITLE
Application for new maintainers?

### ISSUE THREAD

**Issue by marcoceppi** [lipis/bootstrap-social#182]
It's clear this repository has a lot of users but very few updates (3 years!) as branding evolves, new social engagement sites come online, and as time marches on I'd like to propose (prior to hard-forking) adding more maintainers to help keep this project flowing.

To kick things off, I'd like to put in my application; I'm Marco Ceppi an avid open source developer with experience mainly with operations & backend systems. However, I've used and maintain quite a few frontend projects and am well versed in Javascript and CSS.

My goals for this project would be (in addition to maintaining the current list of providers) to help expand how it can be used, including adding the project to exisiting CDN services to allow developers a chance to trial the project before inclusion, making the source directories more modular for each provider, and adding a compilation process that would allow users to strip away providers they didn't want to optimize build sizes.

**Comment by lipis** [comment]
We need to update this library to support Bootstrap 4 to begin with.. do you want to start by creating a PR for the current providers before splitting the rest?

**Comment by marcoceppi** [comment]
Yeah, I think that sounds like a great start. I'll put something up after work later today

**Comment by lipis** [comment]
Thanks man! Much appreciated..

**Comment by YasharF** [comment]
Is PR https://github.com/lipis/bootstrap-social/pull/162 sufficient for Bootstrap 4 support?  I replaced bootstrap-social with @ladjs/bootstrap-social in [Hackathon Starter](https://github.com/sahat/hackathon-starter) over a year ago and it seems to have been working fine.

**Comment by YasharF** [comment]
@lipis @marcoceppi what is the status of this package?  Is it currently considered abandoned and in need of a new owner?
